Account & Project/1. User guides/11. 檢視計畫資訊.md

```diff
@@ -1 +1,67 @@
+---
+title: 檢視計畫資訊 | en
+tags: Guide, Service, EN
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# View project information
+
+
+### Step 1. Sign in TWCC
+
+- Click "Sign in" on the homepage and enter your account password
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_fcf72884bd23c1de1fb70e2d8676b53d.png)
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:** 
+If the screen is blank or abnormal after login, it may be caused by the following problems:
+1. Did not apply for an account, please [<ins>sign up for an account</ins>](https://iservice.nchc.org.tw/nchc_service/nchc_member_apply_1.php)。
+2. Did not join any project that has enabled TWCC service, please [<ins>apply project</ins>](https://man.twcc.ai/@twccdocs/apply-project-and-credit-zh)==**Need to update:exclamation:**== , or notify the tenant admin to add you to the project.
+3. TWCC portal update, please clear the browser cache to sign in normally.
+:::
+
+### Step 2. View project information
+
+After sign in, you can view the project information on the overview page. The information includes:
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_a483ca66912a976c4758b729b7a43cd7.png)
+
+<span>&#10102;</span> User Identity: tenant admin or tenant user, in addition to the title, the screen picture are also different
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:** There are difference betwen tenant admin and users to use TWCC functions, please refer to [<ins>User permissions comparison</ins>](https://man.twcc.ai/@twccdocs/role-main-zh). ==**Need to update:exclamation:**==
+:::
+<span>&#10103;</span> Wallet and credit information, duration, system code
+    
+
+- **Tenant Admin** In addition to the personal credits, you may also view the remaining credit of the project and the total credit of the project.
+- Definition description:
+    - Duration: The effective period of the project.
+    - System code: The code planned in the system. If you encounter a problem, you can provide this information to the customer service staff.
+    - Wallet: When you use the service, the type of wallet for which the credit is deducted: main wallet or sub-wallet.
+        - Main wallet: the total credit of the project deducts the creidt of the sub wallet
```

Comment:
[SUG]
1. creidt is > credits are
2. the total credit > the total credits
3. the sub wallet > sub wallets

ref (credits): https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/useconsolidatedbilling-credits.html